src/routes/jira/events/jira-events-uninstall-post.test.ts

```diff
@@ -75,10 +75,10 @@ describe("Webhook: /events/uninstalled", () => {
 
 	it("Existing Installation, no Subscriptions", async () => {
 		when(jest.mocked(getJiraClient))
-			.calledWith(jiraHost, undefined, undefined, expect.anything())
+			.calledWith(installation.jiraHost, undefined, undefined, expect.anything())
```

Comment:
Also, I don't think `jiraHost` === `installation.jiraHost`, hence not matching either.